### ISSUE TITLE
Issue with java installation and $JAVA_HOME env var setting on Ubuntu 18.04.6 (OSD5)

### ISSUE THREAD

**Issue by ngoan1608** [test-fullautomation/RobotFramework_AIO#162]
Hi Thong,

As the observation on @HolQue 's machine - OSD5:
- Unexpected error log of `apt-get update`  is shown when installing our ***.deb** package.
=> It will also impact to users who are having [Ubuntu Update Error](https://operavps.com/docs/fix-ubuntu-update-error/#:~:text=However%2C%20let's%20see%20what%20are,autoclean%20%2C%20and%20sudo%20apt%20autoremove%20.).
- The `$JAVA_HOME` is not set after installation.

Thank you,
Ngoan

**Comment by huavanthong** [comment]
Hi @ngoan1608 ,
Cc @test-fullautomation , @HolQue 

This issue related to ticket https://github.com/test-fullautomation/RobotFramework_AIO/issues/127
Following the solution in that ticket, I would like to clarify the issue happened following info below:
**Issue 1:** Unexpected error log of apt-get update is shown when installing our *.deb package.
* Because we want to request the user install Open JDK on local machine when they install our RobotFramework AIO.
* To install OpenJDK, we use the command below:
```bash
sudo apt-get update
sudo apt-get install -y default-jdk
```
* Problem: If there are any error happened on the machine by the above command, it will generate the log message as same as Holger's observation.
* Solution: To solve this issue, I would like to update the solution following code:
```bash
 if ! sudo apt-get install -qq default-jdk; then # Hide the console log for installing Open JDK by using option -qq
      echo -e "${MSG_INFO} Unable to install OpenJDK successfully." # If error, notify the user install it maually
      echo "For using PlantUML, please install OpenJDK manually."
      echo "sudo apt-get install -y default-jdk"
 else
      echo -e "${MSG_DONE} default-jdk package has been installed." # Install successfully.
 fi

```


**Issue 2:** The $JAVA_HOME is not set after installation.
* For setting JAVA_HOME to environment on user, it will implement at script [set_robotenv.sh](https://github.com/test-fullautomation/RobotFramework_AIO/blob/develop/config/build/dpkg_build/set_robotenv.sh)
```bash
JDK_PATH=$(update-alternatives --query javac | grep 'Value: ' | grep -o '/.*/bin/java')
```
* Because we have a use case that the user can't install it successfully, or `update-alternatives` command error for checking OpenJDK.
* Therefore, it maybe will not set after installation.

That's all for my info. If you have any concerns, please tell me.

Thank you,   
Thong Hua

**Comment by ngoan1608** [comment]
fixed with merged PR #163 .
This ticket can be closed.
